scripts/src/checkprcontent/checkpr.py

```diff
@@ -140,9 +141,10 @@ def ensure_only_chart_is_modified(api_url, repository, branch):
             else:
                 matches_found += 1
                 if reportpattern.match(file_path):
-                    print("[INFO] Report found")
+                    print(f"[INFO] Report found: {file_path}")
                     print("::set-output name=report-exists::true")
                     report_found = True
+                    report_path = file_path
```

Comment:
I think `report_path` is not needed here. We are not using it anywhere after defining.

Reply:
Good catch, removed.